tools/autograd/derivatives.yaml

```diff
@@ -267,6 +267,9 @@
 - name: cummax(Tensor self, int dim) -> (Tensor values, Tensor indices)
   self: cummax_backward(indices, grad.to(self.scalar_type()), self, dim)
 
+- name: cummin(Tensor self, int dim) -> (Tensor values, Tensor indices)
+  self: cummin_backward(indices, grad.to(self.scalar_type()), self, dim)
```

Comment:
I don't see why this needs the `grad.to(self.scalar_type())` call -- this is probably coming from functions that do type promotion but there's no reason for cummin to do that.

Reply:
yeah we don't need type promotion for cummin and cummax. will update, thank you!